src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -97,10 +101,12 @@ public <CTX> void getQueueStats(CTX mCtx, GetQueueStatsMentor<CTX> mentor) {
                 for (Queue q : req1.queues) req1.queueNames.add(q.name);
                 fetchRetryDetails(req1, (ex2, req2) -> {
                     if (ex2 != null) { onDone.accept(ex2, null); return; }
-                    attachDequeueStats(req2, (ex3, req3) -> {
-                        if (ex3 != null) { onDone.accept(ex3, null); return; }
-                        onDone.accept(null, req3.queues);
-                    });
+                    if (modConfig.getDequeueStatisticReportIntervalSec() > 0){
```

Comment:
Did you test this locally with
```
GET http://localhost:7070/queuing/monitor
```
since it is not working. The result is a pending request when dequeueStats are disabled.